app/src/main/java/com/mapbox/vision/examples/activity/ar/ArMapActivity.kt

```diff
@@ -80,10 +103,7 @@ class ArMapActivity : AppCompatActivity(), MapboxMap.OnMapClickListener,
     override fun onStop() {
         super.onStop()
         mapView.onStop()
-        locationLayerPlugin?.onStop()
-        locationEngine?.removeLocationEngineListener(this)
-        locationEngine?.removeLocationUpdates()
-        locationEngine?.deactivate()
+        arLocationEngine.removeLocationUpdates(locationCallback)
```

Comment:
callback to `arLocationEngine` isn't attached in `onStart`. It's not clear. Will it work when user enable-disable display or go to another app and come back to our one?

Reply:
fixed, call `mapView.getMapAsync(this)` `onStart()`